src/tape/TapeCore.hpp

```diff
@@ -121,6 +122,7 @@ class TapeCore
 		    TAPE_CALL_MAPPING("rgl_node_points_filter_ground", TapeCore::tape_node_points_filter_ground),
 		    TAPE_CALL_MAPPING("rgl_node_points_radar_postprocess", TapeCore::tape_node_points_radar_postprocess),
 		    TAPE_CALL_MAPPING("rgl_node_points_radar_track_objects", TapeCore::tape_node_points_radar_track_objects),
+		    TAPE_CALL_MAPPING("rgl_node_points_radar_set_classes", TapeCore::tape_node_points_radar_set_classes),
```

Comment:
Please add a test to [TapeTest](https://github.com/RobotecAI/RobotecGPULidar/blob/develop/test/src/TapeTest.cpp)